Backend/routes/update-req.js

```diff
@@ -0,0 +1,27 @@
+const ans = require('../db/api_call');
+const helperConflict = require('../helpers/conflict')
+
+
+var updateReq = async (req, res) => {
+    let extention = ""
+    extention = 'update/' + req.params.id

```

Comment:
you dont have to initialize it as an empty string first you can just set this variable below